src/KomaUI.jl

```diff
@@ -221,8 +221,8 @@ function KomaUI(; darkmode=true, frame=true, phantom_mode="2D", sim=Dict{String,
         acq_data = AcquisitionData(raw_aux)
         acq_data.traj[1].circular = false #Removing circular window
         acq_data.traj[1].nodes = acq_data.traj[1].nodes[1:2,:] ./ maximum(2*abs.(acq_data.traj[1].nodes[:])) #Normalize k-space to -.5 to .5 for NUFFT
```

Comment:
Maybe this line is generating problems. 
```suggestion
        acq_data.traj[1].nodes = acq_data.traj[1].nodes[1:3,:] ./ maximum(2*abs.(acq_data.traj[1].nodes[:])) #Normalize k-space to -.5 to .5 for NUFFT
```